cmd/lakefs/cmd/superuser.go

```diff
@@ -42,7 +45,19 @@ var superuserCmd = &cobra.Command{
 			os.Exit(1)
 		}
 
-		authService := auth.NewDBAuthService(dbPool, crypt.NewSecretStore(cfg.GetAuthEncryptionSecret()), cfg.GetAuthCacheConfig(), nil)
+		var authService auth.Service
+		if dbParams.KVEnabled {
```

Comment:
I think that after the 3rd time of this code repeating itself - it might be a good time to create a factory for NewAuthService (consider the solution with the store that @itaiad200 implemented in actions pkg)

Reply:
we decided not to do it